app/src/ui/dispatcher/dispatcher.ts

```diff
@@ -3174,7 +2991,7 @@ export class Dispatcher {
     )
 
     if (result === CherryPickResult.CompletedWithoutError) {
-      this.statsStore.recordCherryPickSuccessfulWithConflicts()
+      this.statsStore.increment('cherryPickSuccessfulCount')
```

Comment:
I think this is the one you were looking for, @niik !
```suggestion
      this.statsStore.increment('cherryPickSuccessfulWithConflictsCount')
```